config/default/kustomization.yaml

```diff
@@ -25,9 +25,9 @@ namePrefix: kubemacpool-
   # Only one of manager_auth_proxy_patch.yaml and
   # manager_prometheus_metrics_patch.yaml should be enabled.
 resources:
-- ../rbac/rbac_role.yaml
-- ../rbac/rbac_role_binding.yaml
-- ../manager/manager.yaml
+- rbac/rbac_role.yaml
```

Comment:
could you please comment on this change in the commit message? or preferably move it to a separate commit?